i18n/locales/co-CO/dashboard/dsls.yml

```diff
@@ -0,0 +1 @@
+co:
```

Comment:
Expected that these are blank? I'm guessing since they're new they have no translations yet?

Reply:
yep!